### PR TITLE
Helm 3 support

### PR DESCRIPTION
**Pull request by tvoran**
- [x] unit tests passing
- [x] acceptance tests passing

### CODE REVIEW

File: test/acceptance/server-ha.bats [hunk 1]
```diff
@@ -95,17 +94,17 @@ setup() {
   kubectl create namespace acceptance
   kubectl config set-context --current --namespace=acceptance
 
-  helm install https://github.com/hashicorp/consul-helm/archive/v0.8.1.tar.gz \
-    --name consul \
+  helm install consul \
+    https://github.com/hashicorp/consul-helm/archive/v0.8.1.tar.gz \
```
Comment (jasonodonnell) [inline test/acceptance/server-ha.bats]: Let's bump this version?

Comment (tvoran) [reply test/acceptance/server-ha.bats]: Sure, bumped to 0.16.2.

**Comment by jasonodonnell** [comment]
@tvoran I think you need to bump the Chart API version to v2 as well

File: Chart.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-apiVersion: v1
+apiVersion: v2
 name: vault
```
Comment (JulienBreux) [inline Chart.yaml]:
```suggestion
name: vault
type: application
```

Comment (tvoran) [reply Chart.yaml]: Thanks for the review! I think we'll leave this out for now since it looks like `application` is the default.

Comment (JulienBreux) [reply Chart.yaml]: 🙏

**Pull request merged**